src/domain/event/occurrences/OccurrencesTable.tsx

```diff
@@ -215,6 +215,12 @@ const OccurrenceEnrolmentTable: React.FC<{
 
       id: 'time',
     },
+    {
+      Header: t('enrolment:occurrenceTable.columnLanguage'),
+      accessor: (row: OccurrenceFieldsFragment) =>
+        row.languages.edges.map((lang) => lang?.node?.id).join(', ') ?? '-',
+      id: 'languages',
+    },
```

Comment:
aria-label should interpret the language in full, not as abbreviations. It was also mentioned in the Jira-ticket.

Reply:
Actually, when tested with a voice over, all the show details buttons are the same and the row is not interpreted at all. I think the row needs to be interpreted by a voice over somehow. Maybe the row content should be included in the buttons aria-label?

Reply:
I can approve this PR without accessibility changes, but an accessibility ticket needs to be created then.